CONTRIBUTING.md

```diff
@@ -142,6 +142,15 @@ cd docs/_build/html/
 open index.html
 ```
 
+## Write integration tests
+1. If adding functionality to a new part of the library, create new file with a class that inherits `IntegrationTestCase` from `tests.integration.integration_test_case` to store all individual tests under (ex: `class TestWallet(IntegrationTestCase)`). Otherwise, add to an existing file.
+2. Create an async (likely) or sync (if specifically using an individual sync client) function that contains logic for the test
+3. For customizability (on async functions only), include the `@test_async_and_sync` decorator to test it on sync/async JsonRpc/Websocket clients, manage import modules, specify only websockets, allow a set number of retries, or use the testnet (vs standalone rippled node). This decorator is used on a large majority of tests to ensure correctness on multiple clients (by default, it tests all 4: `AsyncWebsocket`, `Websocket`, `AsyncJsonRpc`, `JsonRpc`). The few tests that do not utilize this decorator are usually specifically targeting coverage on an individual client.
```

Comment:
```suggestion
3. Include the `@test_async_and_sync` decorator to test against all client types, unless you specifically only want to test with one client. You can also use the decorator to:
       * Limit tests to sync/async only 
       * Limit the number of retries
       * Use Testnet instead of a standalone network
       * Manage import modules
```
* Highlighted the default action item up front (So you only need to read more if that's not what you're trying to do)
* Broke the list into bullets to make it easier to read
* Instead of describing the types of tests that don't use this decorator it's more clear to directly explain the reason they don't use it. (Ex. "If you only want to test one client, don't use this docorator" vs "Tests that only test one client usually don't have the decorator")


Also, what does "Manage import modules" mean?

Reply:
Manage import modules - in the test, we write everything asynchronously. The decorator handles converting everything async function to sync. As a result, you need to pass in the sync functions/modules to import inside the decorator.